### PR TITLE
Added benchmark: print factors of a number

### PR DESCRIPTION
**Pull request by anshumanmohan**

### CODE REVIEW

**Comment by anshumanmohan** [comment]
@sampsyo, I didn't know a good place to give attributions but I'd like to note that I referred to the Ackermann and GCD benchmarks to get a sense of bril formatting and argument passing. Sorry if that was less than kosher; please let me know and I'll go about it differently next time.

**Comment by sampsyo** [comment]
Awesome! No, that's perfectly legit—looking at existing code is a great way to learn how any language works. :smiley: I think it's important to avoid looking at much other code when you're working through an actual compiler algorithm (as you'll do soon) to force yourself to work through it, but figuring out how to write Bril programs really is best done by example.

File: benchmarks/factors.bril [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  print fac;
+  num: int = div num fac;
+  jmp .loopcheck;
+
+.ifno:
+  fac: int = add fac one; 
+  jmp .loopcheck;
+
+.loopend:
+  ret one;
```
Comment (Pat-Lafon) [inline benchmarks/factors.bril:34]: Brilirs rejects this which I believe is correct based on #91. Functions without a return type should not return a value.

Comment (anshumanmohan) [reply benchmarks/factors.bril:34]: Thanks, I think I've resolved this [here](https://github.com/sampsyo/bril/pull/150).

**Pull request merged**